activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/resources/schema/process-extensions-schema.json

```diff
@@ -229,7 +245,7 @@
           ]
         },
         "value": {
-          "type": ["string", "number", "integer", "boolean", "object", "array"]
```

Comment:
We modified this in this PR: https://github.com/Activiti/activiti-cloud-modeling-service/pull/474

We need this not to be a string to allow values in output mappings